core/ai_worker.go

```diff
@@ -549,6 +549,39 @@ func (orch *orchestrator) TextToImage(ctx context.Context, requestID string, req
 	return res.Results, nil
 }
 
+func (orch *orchestrator) LiveVideoToVideo(ctx context.Context, requestID string, req worker.GenLiveVideoToVideoJSONRequestBody) (interface{}, error) {
```

Comment:
I undestand that this is the implementation for the Remote Worker? Have you tested if it works in that setup?

Reply:
I have not yet tested if it works with the remote ai-worker configuration